crypto/asn1/a_time.c

```diff
@@ -463,28 +463,29 @@ int ASN1_TIME_print(BIO *bp, const ASN1_TIME *tm)
     v = (char *)tm->data;
     if (v[l - 1] == 'Z')
         gmt = 1;
-    /*
-     * Try to parse fractional seconds. '14' is the place of
-     * 'fraction point' in a GeneralizedTime string.
-     */
-    if (tm->type == V_ASN1_GENERALIZEDTIME
-        && tm->length >= 15 && v[14] == '.') {
-        f = &v[14];
-        f_len = 1;
-        while (14 + f_len < l && f[f_len] >= '0' && f[f_len] <= '9')
-            ++f_len;
-    }
 
-    if (tm->type == V_ASN1_GENERALIZEDTIME)
+    if (tm->type == V_ASN1_GENERALIZEDTIME) {
```

Comment:
Since `f` and `f_len` aren't used anywhere else, I think their definition should be moved inside this block.